src/Illuminate/Collections/LazyCollection.php

```diff
@@ -1057,6 +1057,17 @@ public function chunk($size)
         });
     }
 
+    /**
+     * Split a collection into a certain number of groups, and fill the first groups completely.
+     *
+     * @param  int  $numberOfGroups
+     * @return static
+     */
+    public function splitIn($numberOfGroups)
+    {
+        return $this->chunk(ceil($this->count() / $numberOfGroups));
```

Comment:
This enumerates the whole collection twice :cry:

Reply:
I'll admit I don't understand the lazy collections as well, so if there's a better way to do this, definitely open to it.